x-pack/plugins/uptime/public/pages/mapping_error.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import { useBreadcrumbs } from '../hooks/use_breadcrumbs';
+import { useTrackPageview } from '../../../observability/public';
+
+export const MappingErrorPage = () => {
+  useTrackPageview({ app: 'uptime', path: 'mapping-error' });
+  useTrackPageview({ app: 'uptime', path: 'mapping-error', delay: 15000 });
+
+  useBreadcrumbs([
+    {
+      text: i18n.translate('xpack.uptime.mappingErrorRoute.breadcrumb', {
+        defaultMessage: 'Mapping error',
+      }),
+    },
+  ]);
+
+  return (
+    <EuiEmptyPrompt
+      iconColor="danger"
+      iconType="cross"
+      // TODO: placeholder copy
+      title={<div>Heartbeat mappings are not installed</div>}
+      body={<div>You need to stop Heartbeat, delete your indices, and restart Heartbeat.</div>}
```

Comment:
For copy WDYT of:

> Incorrect mappings detected! Perhaps you forgot to run the heartbeat `setup` command? See the [heartbeat quickstart](https://www.elastic.co/guide/en/beats/heartbeat/current/heartbeat-installation-configuration.html) for more information.

I'm also thinking maybe we should add a dedicated docs page for this and link to that. Thoughts?

Reply:
I've opened https://github.com/elastic/observability-docs/issues/1018 to track the docs

Reply:
i think having inline docs is always amazing, can we also keep instructions inline preferabbly with commands to run and lead to details step by step docs page,

Reply:
Docs PR: https://github.com/elastic/observability-docs/pull/1046